src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilterTest.java

```diff
@@ -179,4 +180,79 @@ private static SuppressionXpathFilter createSuppressionXpathFilter(String fileNa
         return suppressionXpathFilter;
     }
 
+    private static LocalizedMessage getMessageForStringLiteral(int lineNo, int columnNo) {
+        return new LocalizedMessage(lineNo, columnNo, TokenTypes.PLUS, "", "",
+                null, null, "777", SuppressionXpathFilterTest.class, null);
+    }
+
+    private static LocalizedMessage getMessageForCharLiteral(int lineNo, int columnNo) {
+        return new LocalizedMessage(lineNo, columnNo, TokenTypes.CHAR_LITERAL, "", "",
+                null, null, "777", SuppressionXpathFilterTest.class, null);
+    }
+
+    private static TreeWalkerAuditEvent createTreeWalkerAudit(int lineNo, int columnNo,
+                   int tokenTypes, File file)
+            throws IOException, CheckstyleException {
+        final LocalizedMessage message;
+        if (tokenTypes == TokenTypes.CHAR_LITERAL) {
+            message = getMessageForCharLiteral(lineNo, columnNo);
+        }
+        else {
+            message = getMessageForStringLiteral(lineNo, columnNo);
```

Comment:
no need for methods and if-else block here. Just
```
final LocalizedMessage message = new LocalizedMessage(lineNo, columnNo, tokenTypes, "", "",
                null, null, "777", SuppressionXpathFilterTest.class, null);
```
is enough.
`getMessageFor...` methods can be removed

Reply:
Done

Reply:
Not done. There is a method left `getMessageForStringLiteral`, it is not needed. Please inline it.

Reply:
Sorry, I misinterpreted it. It`s done now. Also, I had removed the method, but I had not totally inlined it inside TreeWalkerAuditEvenet as it looks cluttered a lot. So, rather initialised it inside a message variable and then used that.